login/src/main/java/com/mito/login/ui/WelcomeScreen.kt

```diff
@@ -36,49 +50,143 @@ import com.mito.common.navigation.NavigationReferences
 import com.mito.common.navigation.NavigationReferences.ProfileReference.getRoute
 import com.mito.core.navigation.Screen
 import com.mito.login.R
+import kotlinx.coroutines.delay
 
 class WelcomeScreen : Screen {
     override val route: String = NavigationReferences.WelcomeReference.getRoute()
 
     @Composable
     override fun Content(navController: NavHostController) {
         val viewModel = hiltViewModel<WelcomeViewModel>()
+
         WelcomeScreen(navController, viewModel)
     }
 }
 
 @Composable
-fun WelcomeScreen(navController: NavHostController, viewModel: WelcomeViewModel) {
-    Card(
-        modifier = Modifier.fillMaxSize(),
-        colors = CardDefaults.cardColors(Color(0x7CC4F0B8)),
-        elevation = CardDefaults.elevatedCardElevation(6.dp),
-        shape = RoundedCornerShape(25.dp),
+fun WelcomeScreen(navController: NavHostController,viewModel: WelcomeViewModel) {
+
+    var isAnimated by rememberSaveable { mutableStateOf(false) }
+    var cardActivated by rememberSaveable { mutableStateOf(false) }
+    var buttonActivated by rememberSaveable { mutableStateOf(false) }
+    var buttonEnabled by rememberSaveable { mutableStateOf(false) }
```

Comment:
All these values are mutables booleans. Then we can do it using viewmodel.status like LoginScreen